packages/forms/src/styled/tiles/StyledTile.ts

```diff
@@ -153,6 +153,7 @@ export const StyledTile = styled.label.attrs<IStyledTileProps>(props => ({
   cursor: ${props => !props.isDisabled && 'pointer'};
   padding: ${props => props.theme.space.base * 5}px;
   direction: ${props => props.theme.rtl && 'rtl'};
+  min-width: ${props => props.theme.space.base * 24}px;
```

Comment:
At these larger dimensions, the value of referencing `theme.space.base` becomes negligible. For example, we don't want to see this up to `120px` if a user overrides with base-5. In most other places, we just express the value explicitly.

Reply:
For sure, will update this with an explicit value.